shared/api_archive/archive.py

```diff
@@ -33,61 +30,44 @@ class MinioEndpoints(Enum):
     static_analysis_single_file = (
         "{version}/repos/{repo_hash}/static_analysis/files/{location}"
     )
-
     test_results = "test_results/v1/raw/{date}/{repo_hash}/{commit_sha}/{uploadid}.txt"
 
     def get_path(self, **kwaargs):
         return self.value.format(**kwaargs)
 
 
-# Service class for performing archive operations. Meant to work against the
-# underlying StorageService
 class ArchiveService(object):
     """
-    The root level of the archive. In s3 terms,
-    this would be the name of the bucket
+    Service class for performing archive operations.
+    Meant to work against the underlying `StorageService`.
     """
 
-    root = None
-
+    root: str
     """
-    Region where the storage is located.
+    The root level of the archive.
+    In s3 terms, this would be the name of the bucket
     """
-    region = None
```

Comment:
do we care that some self-hosted customer might be using this option or is it never used in the code?

Reply:
Maybe this is a setting for the underlying `StorageService`, idk

Reply:
I haven’t found any code that actually references this, thats why I removed it.